client/src/pages/AdminDashboard.js

```diff
@@ -149,65 +191,85 @@ const AdminDashboard = (props) => {
     setAvgHoursToChart(totalVolunteerAvgHoursByEventType);
   }
 
-  function extractVolunteersSignedInByProperty(events, users, uniqueTypes, propName){
-    let result = {};
+  function extractVolunteersSignedInByProperty(
+    events,
+    users,
+    uniqueTypes,
+    propName
+  ) {
+    const result = {};
     let type;
 
-    uniqueTypes.forEach(el => result[el] = parseInt('0'));
-    for (let eventId of users.keys()) {
-      if(propName === 'eventType'){
+    uniqueTypes.forEach((el) => (result[el] = parseInt('0')));
+    for (const eventId of users.keys()) {
+      if (propName === 'eventType') {
         type = events.get(eventId).eventType;
         result[type] = users.get(eventId).length + result[type];
       }
 
-      if(propName === 'hacknight' && typeof events.get(eventId).hacknight !== 'undefined'){
+      if (
+        propName === 'hacknight' &&
+        typeof events.get(eventId).hacknight !== 'undefined'
+      ) {
         type = events.get(eventId).hacknight;
         result[type] = users.get(eventId).length + result[type];
       }
     }
     return result;
   }
 
-  function findTotalVolunteerHours(events, users, uniqueTypes, propName){
-    let result = {};
+  function findTotalVolunteerHours(events, users, uniqueTypes, propName) {
+    const result = {};
     let type;
-    uniqueTypes.forEach(el => result[el] = parseInt('0'));
+    uniqueTypes.forEach((el) => (result[el] = parseInt('0')));
 
-    for (let eventId of users.keys()) {
-      if(propName === 'eventType'){
+    for (const eventId of users.keys()) {
+      if (propName === 'eventType') {
         type = events.get(eventId).eventType;
-        result[type] = result[type] + (events.get(eventId).hours * users.get(eventId).length);
+        result[type] =
+          result[type] + events.get(eventId).hours * users.get(eventId).length;
       }
 
-      if (propName === 'hacknight' && typeof events.get(eventId).hacknight !== 'undefined'){
+      if (
+        propName === 'hacknight' &&
+        typeof events.get(eventId).hacknight !== 'undefined'
+      ) {
         type = events.get(eventId).hacknight;
-        result[type] = result[type] + (events.get(eventId).hours * users.get(eventId).length);
+        result[type] =
+          result[type] + events.get(eventId).hours * users.get(eventId).length;
       }
     }
     return result;
   }
 
-  function findAverageVolunteerHours(totalVolunteers, totalVolunteerHours, uniqueTypes){
-    let result = {};
-    uniqueTypes.forEach(el => result[el] = parseInt('0'));
-
-    for(let eventType of uniqueTypes.keys()){
-      let hours = totalVolunteerHours[eventType];
-      let volunteers = totalVolunteers[eventType];
-      let averageHours = (hours / volunteers);
-      if(!Number.isInteger(averageHours)) averageHours = +averageHours.toFixed(2);
-      volunteers > 0 ? result[eventType] = averageHours : result[eventType] = 0;
+  function findAverageVolunteerHours(
```

Comment:
lines less than 100 symbols shouldn't be formatted

Reply:
(see above)